tests/custom_attribute.rs

```diff
@@ -0,0 +1,86 @@
+// This is used to make sure our dependencies are picked up by cargo winrt
+import!(
+    dependencies
+        nuget: Microsoft.Windows.SDK.Contracts
+        nuget: KennyKerr.Windows.TestWinRT
+    types
+        test_component::*
+        windows::foundation::*
+);
```

Comment:
I just noticed the `package.metadata.winrt.dependencies` section in the root `Cargo.toml` file. I'm guessing that is what is informing cargo winrt of the dependencies? If so, I'll delete this block.

Reply:
Yup, looks like this isn't needed.